example/demo/src/app.rs

```diff
@@ -147,30 +141,23 @@ fn HomePage() -> impl IntoView {
                     <p>a library designed to declaratively pair your keybindings with callbacks.</p>
                 </div>
                 <div class="flex-1 flex flex-col space-y-20">
-                    <div>
-                        <ul>
-                            <li>scope_a + ref_a => counter++ </li>
-                            <li>scope_b + ref_a => does nothing </li>
-                            <li>scope_b + ref_b => counter-- </li>
-                            <li>scope_a + ref_b => does nothing </li>
-                        </ul>
-                    </div>
-                    <div>
-                        <p>Press 's' to toggle between scopes a and b</p>
-                        <p>Click on either div to toggle between ref a and b</p>
-                        <p>Current scope: {move || { current_scope.get() }}</p>
-                        <p>Current ref(ONLY CHANGES IF TRIGGERED): {move || { current_ref.get() }}</p>
+                    <div class="flex space-x-8 items-center">
+
+                        <div>
+                            <p>Press 's' to toggle between scopes a and b</p>
+                            <p>Click on either div to toggle between ref a and b</p>
+                            <p>Current scope: {move || { current_scope.get() }}</p>
+                        </div>
                     </div>
                     <div class="flex-1 grid grid-col-1 lg:grid-cols-2">
                         <div
                             id="scope_a"
                             class:active=move || current_scope.get() == "scope_a"
                             class:not-active=move || current_scope.get() != "scope_a"
-                            _ref={a_ref} 
-                            tabIndex={-1}
+                            tabIndex=-1
```

Comment:
notice how `tabIndex=-1` one is here. This should be in the div where `_ref` attribute is...